app.js

```diff
@@ -34,23 +34,55 @@ function resetStats() {
 }
 
 form.addEventListener('submit', (e) => {
+
     // Step 1 - add code to track each submission to the stats
-    // Create an object to "model" you statistic like the following:
+    //prevents form from refreshing page
+    e.preventDefault();
+    // using FormData built in function to retrieve data from our form
+    const formData = new FormData(form);
+    // Create an object to "model" your statistic like the following:
+    const stat = {
+        player: formData.get('player'),
+        points: formData.get('points')
```

Comment:
You can set the points to a number data type as it comes into your app. You can do this by wrapping `formData.get('points')` in the `Number()` function.